### PR TITLE
Prepare for 6.6.1 Release

### PR DESCRIPTION
**Pull request by azeey**
# 🎈 Release

Preparation for 6.6.1 release.

Comparison to 6.6.0: https://github.com/gazebosim/gz-rendering/compare/ignition-rendering6_6.6.0...ign-rendering6

Needed by <PR(s)>

## Checklist
- [ ] Asked team if this is a good time for a release
- [ ] There are no changes to be ported from the previous major version
- [ ] No PRs targeted at this major version are close to getting in
- [ ] Bumped minor for new features, patch for bug fixes
- [ ] Updated changelog
- [ ] Updated migration guide (as needed)
- [ ] Link to PR updating dependency versions in appropriate repository in [gazebo-release](https://github.com/gazebo-release) (as needed): <LINK>

**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -1,5 +1,28 @@
 ## Ignition Rendering
 
+### Ignition Rendering 6.6.1 (2023-09-01)
+
+1. Fixed light visual in OGRE
+    * [Pull request #864](https://github.com/gazebosim/gz-rendering/pull/#864)
```
Comment (iche033) [inline Changelog.md:6]:
```suggestion
    * [Pull request #864](https://github.com/gazebosim/gz-rendering/pull/864)
```

**Comment by azeey** [comment]
It looks like macOS has 33 failures on Monterey, but 0 on Big Sur.

File: Changelog.md (hunk 1 above)
Comment (azeey) [reply Changelog.md]: Done in https://github.com/gazebosim/gz-rendering/pull/891/commits/4747722e17a13b48977e4d9835af3fd49d461689

**Comment by scpeters** [comment]
> It looks like macOS has 33 failures on Monterey, but 0 on Big Sur.

I can't tell what the problem was from the console log; it looks like those tests have been failing for a while on `gz-rendering6` CI, since it often seems to run on the monterey machine:

* https://build.osrfoundation.org/view/ign-fortress/job/ignition_rendering-ci-ign-rendering6-homebrew-amd64/

I would just go ahead with the release I think

**Comment by azeey** [comment]
I also can't figure out why there are 20 warnings on windows. Going ahead with the release.

**Pull request merged**